content/blog/2023-01-31-interview-open-flexure-microscope/index.en.md

```diff
@@ -0,0 +1,791 @@
+We've got a custom motor driver there, we are trying to get the latest
+version of this out. We've had some issues getting it released as I don't do electronics (or at least not the advanced surface mount stuff). The board was designed by Valerian in Tanzania. We've also got a small illumination circuit board. It turns out that one of the biggest, costs of building a microscope is free hand soldering wires and the LED together as it's actually quite a time consuming process. And of course, time is money in production. Making a custom little board, which is basically a
+constant current driver and a resistor, will save you a lot of money in
+the long run. As for the mechanical components, there are quite a few. But most
+of the mechanism is in one single part: the main body. Then there are
+peripherals, where we've got some gears, somewhere to mount the illumination, somewhere to mount the objective and, a little base.
+
+Then there is the software. A huge amount of effort went into the
+software. Most of this was done by Joel, a previous postdoc. We
+run a client server architecture. On the microscope there is a server
+that deals with all of the interaction with all the different pieces of
+hardware. That server has an API. There's a client interface, which can
+communicate with the microscope. You can either run it the same computer that is in the microscope (a little embedded Raspberry Pi) or you can run it
+externally over a network. The nice thing about having the server client
+interface is, scientists can use it and automate it from MATLAB or
+Python. We've also got MATLAB and Python clients that can talk to the
+API. The software, I think, is very mature for an open hardware project,
+especially a science open hardware project. And I think even if
+you consider a lot of very expensive scientific instruments, the
+user interface of the Software is often an afterthought. Even for
+professional instruments the UI is often terrible. Ours is at least reasonably good. There's a lot of things we need to fix, but it's actually quite reasonable.
+{{< /expand >}}
+
+> And has this system been built or produced by others independently?
+
+{{< expand "Show answer">}}
+It's very hard to know how many people have built it. You only know the people
+who've reported back and told you they've built it. So we did a little
+sort of survey asking on Twitter and on our forum. Have you built it? And where
+have you built it or used it? So far, it's been used on every continent,
+including Antarctica. We've got a photo of somebody using it on the ice
+in Antarctica. It's been built in about 40 countries. We also have a user forum with over
+250 people. We don't know if every single one of them has built or used
+the microscope, some people will have joined but turn off "ask questions". But the community we know about may just be the tip of the iceberg, we assume there are thousands of microscopes out there,
```

Comment:
```suggestion
the microscope, some people will join only to ask questions. But the community we know about may just be the tip of the iceberg, we assume there are thousands of microscopes out there,
```

Reply:
I think "will have joined only to ask questions" is most likley what I said/meant to say